### ISSUE TITLE
Installer doesn't work anymore on Ubuntu 20.04

### ISSUE THREAD

**Issue by Hornochs** [saveriomiroddi/zfs-installer#191]
Heyho, 
so I tried at first this installer with Kubuntu - while the script run complety, the installer didn't started and didn't picked up the zfs pool.

Then I tried it with Ubuntu Desktop 20.04, Script fails at the beginning, telling me, "dialog" has no installation candidate. So I'm stuck at the moment. :/

**Comment by saveriomiroddi** [comment]
Hi there! There have been very significant changes. That package is intended to be installed. I'll check it today and get back to you :smile:

**Comment by saveriomiroddi** [comment]
Hmmm. Quite confusing. I've tested on Ubuntu MATE 20.04.1, and it works (it's my standard test image). I'm going to test on the latest Ubuntu 20.04 standard.

**Comment by saveriomiroddi** [comment]
This is caused by the Ubuntu Desktop standard not enabling the `universe` repository by default.

It's an easy fix, I'll take care of this in a few hours.

**Comment by Hornochs** [comment]
Muhaha :D Thank you :)

Any advice about the Kubuntu Installer? I mean I can make therefore another issue :) Reason is I want to use Kubuntu, but don't want to install that all manually after seeing this Project :D

**Comment by saveriomiroddi** [comment]
Unfortunately, that's an issue I will need to look separately into, and that is for sure not so easy.

Something you can try is the previous installer version, and see what happens:

```
GET https://raw.githubusercontent.com/saveriomiroddi/zfs-installer/0.3.14/install-zfs.sh | sudo bash
```

**Comment by saveriomiroddi** [comment]
Fix released with v0.5.1.
